index.html

```diff
@@ -196,6 +220,7 @@ <h1 class="card-title">Multi-Post Sorties</h1>
     </section>
 
     <section class="about-me">
+    <div class="me">
         <h1 class="card-title">About</h1>
         <h1 class="card-title">Myself</h1>
 
```

Comment:
I will be suggesting some changes in the `About Myself` section. But first, let's take a look at what we're having at the moment compared to the Figma design:
_**Figma Design**_
![original](https://user-images.githubusercontent.com/26856491/121509419-87035f80-c9de-11eb-9be3-5a29e5d89873.png)

**_Your Design_**
![clone](https://user-images.githubusercontent.com/26856491/121509428-89fe5000-c9de-11eb-865a-bfdb724262a6.png)

From the images below, I think the following changes should be made: 
- We have to ensure that the left block takes 40% and the right takes the remaining 60% of the page's width
- We should have equal margins to the left and right sides of the whole section.
- The dropdown carets should be aligned towards the right side of each skillset (languages, frameworks, skills).
- There should be some space between the elements under the `Languages` skill.
- It is also important that you use the same typography (font style, weight, etc) as observed in the figma design.

Reply:
Well addressed this👍